src/app/shared/ingredient.model.ts

```diff
@@ -1,3 +1,3 @@
 export class Ingredient {
```

Comment:
Do you need additional level: 'models' folder to store all items of same type in one place?